README.md

```diff
@@ -13,13 +13,14 @@ View icons from bootstrap's icon library and insert them into your code through
 - Users can search for the icon-name (or tags associated with it) and choose the desired icon
 - On clicking on the icon, corresponding svg code will be inserted at the last active position on the code editor
 - Users can change the height and width of the svg directly from the side panel
-![demo](https://github.com/Open-Sourced-Olaf/Get-svg-icons/blob/staging/screenshots/side-panel.gif)
+![demo](https://raw.githubusercontent.com/anjalisoni3655/Get-svg-icons/staging/screenshots/side-panel.gif)
```

Comment:
https://raw.githubusercontent.com/Open-Sourced-Olaf/Get-svg-icons/staging/screenshots/side-panel.gif
Can you use this link instead? I think the one you are using now is from your fork.

Reply:
Yeah, didn't notice that. Please have a look, it should be done now.